### PR TITLE
Aggressively delete a cluster template if it exists.

### PR DESCRIPTION
**Pull request by brendandburns**
@mbforbes @ixdy @quinton-hoole

### CODE REVIEW

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -401,8 +401,11 @@ function create-node-template {
   #                 add retries. Just relying on a non-zero exit code doesn't
   #                 distinguish an ephemeral failed call from a "not-exists".
   if gcloud compute instance-templates describe "$1" --project "${PROJECT}" &>/dev/null; then
-    echo "Instance template ${1} already exists; continuing." >&2
-    return
+    echo "Instance template ${1} already exists; deleting." >&2
+    if ! gcloud compute instance-template delete "$1" --project "${PROJECT}" &>/dev/null; then
+      echo -e "${color_yellow}Failed to delete existing instance template" >&2
```
Comment (mbforbes) [inline cluster/gce/util.sh]: This should end with `${color_norm}`

File: cluster/gce/util.sh [hunk 2]
```diff
@@ -401,8 +401,11 @@ function create-node-template {
...
+      exit 2
```
Comment (mbforbes) [inline cluster/gce/util.sh]: `exit` seems a bit intense but I guess it will just fail afterwards, so makes sense

**Comment by mbforbes** [comment]
LGTM w/ nits

File: cluster/gce/util.sh (hunk 1 above)
Comment (brendandburns) [inline cluster/gce/util.sh]: done.

File: cluster/gce/util.sh (hunk 2 above)
Comment (brendandburns) [inline cluster/gce/util.sh]: well, the alternative is turning up a borked cluster, I think I prefer exit.

**Comment by brendandburns** [comment]
Comments addressed, ptal.

**Comment by brendandburns** [comment]
I'm going to merge this, as this problem is currently breaking a number of e2e runs.

**Comment by mbforbes** [comment]
Sorry for not getting back, but was truly LGTM. I'll add the label for posterity :-)

**Comment by quinton-hoole** [comment]
Root cause #9016. More fixes to come.

**Pull request merged**